### DISCUSSION TITLE
How to make the hidden GPU Tempreture Target show up ?

### DISCUSSION THREAD

**Discussion by Feeeg** [seerge/g-helper#2353] [Q&A]
My NVIDIA VBIOS shows that it can be configured with a temperature limit between 75 to 87 degrees, but Ghelper does not have this option. How can I edit the JSON file to bring out the the hidden option?
![批注 2024-04-01 191759](https://github.com/seerge/g-helper/assets/165661648/8209e11d-e0f8-44f8-82b3-abe2c03c3485)

**Comment by seerge** [comment]
@Feeeg if you don’t see this slider in the app it means your model doesn’t support it. It needs to be implemented on BIOS level by Asus to be supported

**Reply by Feeeg** [reply]
I can set a temperature limit between 75 to 87 using  ASUS's overclocking tool GPU tweak III ,and it works but the downside of this tool  is that I have to reconfigure it after each reboot. From this perspective, it seems to be supportive.

**Reply by seerge** [reply]
What laptop model do you have? G-Helper looks at special endpoints that asus has exposed for setting each value. If endpoint doesn't exist - it hides options from UI.

**Reply by Feeeg** [reply]
FA506iu

**Reply by seerge** [reply]
This is an old model, and those settings don't exist there for sure. I dont know how exactly GPU Tweak is doing that. With a good chance it uses either ``nvidia-smi`` tool under the hood or something like that . Anyway, G-Helper relies only on "native" way to set that, so it can't be added :)

**Reply by Feeeg** [reply]
Still, I must say a big thank you to you
